### DISCUSSION TITLE
Chameleon Light/Dark Themes

### DISCUSSION THREAD

**Discussion by DavidFuchs** [TriliumNext/Trilium#6026] [Show and tell]
Hey there. I created light and dark themes (dubbed Chameleon) for Trilium a while back and wanted to share them here for possible addition to the Theme Gallery. I think they were on a wiki somewhere before the project became Trilium Next.

https://github.com/DavidFuchs/trilium-chameleon-theme

The accent color is customizable with a single variable (--palette-accent-hue), making it easy for end-users to easily customize to their preference (hence the 'chameleon' name).

Thanks!

**Comment by meichthys** [comment]
For reference: https://triliumnext.github.io/Docs/Wiki/theme-gallery.html

**Comment by Engr-AllanG** [comment]
Is anyone else having issues with themes applying correctly? I just tried out this Chameleon theme and I get a messed up light version. None of my older themes seem to work either, they have similar white-out appearance

**Reply by Engr-AllanG** [reply]
Disregard. Somehow I had pasted in an error.

**Reply by Engr-AllanG** [reply]
@DavidFuchs Thanks for the cool theme. I really like it. The teal inspired me to play around with it to match logseq, which I have always really liked. I posted mine [here](https://github.com/Engr-AllanG/trilium-themes/tree/master)

**Comment by CobriMediaJulien** [comment]
@DavidFuchs Thank your for your themes! They look great!
